src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar">
```

Comment:
you need to create modifiers for the calendar start-day and month-length
```suggestion
    <div class="calendar calendar--start-day_tue calendar--month-length_29">
```